spec/requests/followships_request_spec.rb

```diff
@@ -1,5 +1,117 @@
-require 'rails_helper'
+require "rails_helper"
+require "shared_contexts"
 
 RSpec.describe "Followships", type: :request do
+  include_context "api request authentication helper methods"
+  include_context "api request global before and after hooks"
 
+  let!(:user) { create(:user) }
+  let(:user_second) { create(:user) }
+  let(:user_third) { create(:user).as_json }
+  let!(:tweet) { create(:tweet) }
+  let!(:follow) { create(:follow, follower_user_id: user.id, following_user_id: user_second.id) }
+
+  before(:each) do
+    @current_user = :user
+    login_as(user, scope: :user)
+  end
+
+  describe "GET /following_users" do
+    it "renders a successful response" do
+      get following_users_url
```

Comment:
El llamado a la url puedes hacerlo mejor en un before para qeu lo peudas usar en caso que tengas varios it que lo utilicen.